src/routing/Page.tsx

```diff
@@ -14,58 +19,66 @@ export default function Page(props: PageProps): JSX.Element {
   const editPage = searchParams.get('edit');
   const sourceAnchor = searchParams.get('source');
+  const state = useFormikContext();
+  const currentLocation = useLocation();
 
   const { nextRoute, previousRoute } = useContext(RouterContext);
 
+  useEffect(() => {
+    // Reset the form on route change.
+    // This could be used in a different component
+    // but was used here because it's related to routing.
+    state.setErrors({});
+    state.setTouched({}); // resetting fields
+    state.setSubmitting(false); // resetting fields
+  }, [currentLocation]);
+
   return (
     <div>
       <h3>{props.title}</h3>
-      <Form>
-        <div className="vads-u-margin-y--2">
-          {props.children}
-        </div>
+      <div className="vads-u-margin-y--2">{props.children}</div>
 
-        {editPage && (
-          <div>
-            <button
-              onClick={(event) => {
-                event.preventDefault();
-                navigate(
-                  `/review-and-submit${
-                    sourceAnchor ? `#${sourceAnchor}` : ''
-                  }` as To
-                );
-              }}
-              className="btn next"
-            >
-              Back to Review page
-            </button>
-          </div>
-        )}
-        {previousRoute && !props.hidePreviousButton && (
+      {editPage && (
+        <div>
           <button
-            className="btn usa-button-secondary prev"
             onClick={(event) => {
-              event.preventDefault();
-              navigate(previousRoute as To);
+              navigate(
+                `/review-and-submit${
+                  sourceAnchor ? `#${sourceAnchor}` : ''
+                }` as To
+              );
             }}
+            className="btn next"
           >
-            <i className="fas fa-angle-double-left"/> Previous
+            Back to Review page
           </button>
-        )}
+        </div>
+      )}
+      {previousRoute && !props.hidePreviousButton && (
+        <button
+          className="btn usa-button-secondary prev"
+          onClick={() => {
+            navigate(previousRoute as To);
+          }}
+        >
+          <i className="fas fa-angle-double-left" /> Previous
+        </button>
+      )}
 
-        {nextRoute && (
-          <button
-            className="btn usa-button-primary next"
-            onClick={(event) => {
-              event.preventDefault();
-              navigate(nextRoute as To);
-            }}
-          >
-            {props.nextButtonCustomText ? props.nextButtonCustomText : 'Next'} <i className="fas fa-angle-double-right"/>
-          </button>
-        )}
-      </Form>
+      {nextRoute && (
+        <button
+          className="btn usa-button-primary next"
+          onClick={(event) => {
+            state.handleSubmit();
+            navigate(nextRoute as To);
+          }}
+          type="submit"
+          disabled={Object.keys(state.errors).length > 0}
```

Comment:
Thanks for the feedback. I was adding this to test the functionality but didn't think to switch it back. I will look at erroring the page on button click.

Reply:
I think there might be something weird going on with how our `TextField` component is rendering required errors. They are missing `role="alert"` which might be causing VoiceOver to skip over reading the error messages.

Reply:
Okay, I just altered it so it throws up errors if you try to submit. Would this be the intended functionality?

Reply:
Yes. Change looks great, thanks @mlauren. PR looks fantastic as well. I'll create a bug ticket for the VoiceOver text field issue, since it's unrelated to your work here.

Reply:
@micahchiang is this something that's missing on the design system side or should be added to the input field? When I tab past the field in voiceover, it is reading "please provide a response" but it is the last thing that is read.

Reply:
@mlauren - which field are you getting that on? In Firefox it isn't reading the error message for first or last name, but reads it for the radio group component on `ClaimantInformation`